concent_api/concent_api/system_check.py

```diff
@@ -719,3 +735,18 @@ def check_use_signing_service(app_configs: None=None, **kwargs: Any) -> list:  #
             return [create_error_55_use_signing_service_is_true_but_middleman_is_missing()]
 
     return []
+
+
+@register()
+def check_gntdeposit_adress(app_configs: None=None, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    if (
+        hasattr(settings, 'PAYMENT_BACKEND') and
+        settings.PAYMENT_BACKEND == 'core.payments.backends.sci_backend'
+    ):
+        if not hasattr(settings, 'GNT_DEPOSIT'):
+            return [create_error_56_gntdeposit_not_set()]
+
+        if not isinstance(settings.GNT_DEPOSIT, str):
+            return [create_error_57_gntdeposit_has_wrong_type(settings.GNT_DEPOSIT)]
```

Comment:
Does this address have e.g permanent length which can be checked? Or something more that is important..?

Reply:
Actually yes. It's like every address on ethereum blockchain, it starts with `0x` and should have length of 42 characters(I need to double check that though).

Reply:
Ok, I've added value check after type check.